services/storage/tests/unit/test_dsm_dsmcleaner.py

```diff
@@ -117,6 +118,7 @@ async def test_clean_expired_uploads_deletes_expired_pending_uploads(
         file_or_directory_id,
         link_type,
         file_size,
+        sha256_checksum=faker.sha256(),
```

Comment:
I would like to also have a test when this is not provided. Please add a parameter to the test with None or a checksum.

Reply:
My two cents here, if you think it is important to test for having or not having the checksum. Either you add it to all tests in this module (and also provide the None option) or you provide None to all of them.

Reply:
Good point. I will add more cases to the tests by parametrizing them

Reply:
I have a slight improvement on your version here. Please consider using that one. Closing this thread.

Reply:
Yep, I added your improvements, thanks